src/Network/Network/PrivateLinkService/PrivateEndpointConnection/PrivateEndpointConnectionBaseCmdlet.cs

```diff
@@ -76,6 +77,8 @@ public new object GetDynamicParameters()
 
         protected IPrivateLinkProvider BuildProvider(string subscription, string privateLinkResourceType)
         {
+            if (!GenericProvider.SupportsPrivateLinkResourceType(privateLinkResourceType))
+                throw new System.Exception($"The {privateLinkResourceType} doesn't support private endpoint connection");
```

Comment:
```suggestion
                throw new System.Exception($"The {privateLinkResourceType} doesn't register private endpoint connection in current configuration");
```
The exception type should be AzPSApplicationException.
Please put the exception message into Properties.Resources.xxx" as the message is used many times.

Reply:
Done